examples/generic-azure.yaml

```diff
@@ -672,4 +672,39 @@ groups:
 
           ## References
           - section 9.1.2 https://documentation.suse.com/sbp/all/single-html/SLES4SAP-hana-sr-guide-PerfOpt-15/
+      - id: 1.5.3
+        description: "ntp clock synchronization is working and using the slew mode"
+        audit: |
+          # The third field of ntpq -p is the stratum and if it's 16
+          # the sync is disabled. Make sure it's not 16
+          if ! sudo ntpq -pn | grep "No association ID's returned" \
+          && sudo ntpq -pn | awk '{print $3}' | grep -e "[0-9]" | grep -v 16; then
```

Comment:
stratum of 16 is one indicator of why a ntp source isn't used, there can be others (Ilike too high jitter). I really suggest to check for sync flag ('*') instead, like previously mentioned.

Reply:
> stratum of 16 is one indicator of why a ntp source isn't used, there can be others (Ilike too high jitter). I really suggest to check for sync flag ('*') instead, like previously mentioned.

@dirkmueller thank you for the feedback. I have strengthened the grep expression and grep only the ntp servers with the '* flag'